src/gsi/apply_scaledepwgts.f90

```diff
@@ -0,0 +1,151 @@
+!$$$  subprogram documentation block
+!28-03-2018    T. Lei and D. Kleist   consoliated and added codes 
+!                                     for the scale dependent scale localization scheme
+ function fwgtofwvlen (rvlft,rvrgt,rcons,rlen,rinput)
```

Comment:
For this file, please:
Add intent attribute to all arguments. 
Adjust space indentation to make it consistent with other code.
Use 3 space indentation for all do and if loops
explicitly specify precision of local numeric constants

Reply:
@Wangy1111 , Please allow me as the question here,  why is the line indicating the 28-03-2018 , T. Lei and D. Kleist" 's coding deleted?    OU 's sdl coding is from the global SDL (as in the 2021 paper with Huang, Wang, Lei, Kleist  as the four authors.  while EMC made major contribution to then global SDL , which was also the starting frame for OU's following coding development.  Though we at EMC didn't leave many "authorship claims" in the codes as required . But I definitely hope to know the rare one I left would be deleted now.  Thanks for your clarification.

Reply:
@Wangy1111  Thanks.